lib/open_project/text_formatting/matchers/attribute_macros.rb

```diff
@@ -50,14 +52,29 @@ def self.applicable?(content)
         content.include?("Label:") || content.include?("Value:")
       end
 
-      def self.process_match(m, _matched_string, _context)
+      def self.work_package_context?(context)
+        context[:object].is_a?(WorkPackage) || context[:object].is_a?(API::V3::WorkPackages::WorkPackageEagerLoadingWrapper)
```

Comment:
The check against EagerLoadingWrapper is oddly specific and will likely break, especially given that there's no spec for this case. We could try passing in a `resource` explicitly to the formatter, which would either be the work package or project to be rendered for.

Reply:
I agree and I was suprised that context[:object].is_a?(WorkPackage) was not enough.

Still I need to know what class in context[:object] as this could be a wiki page or something totally different with an ID, so having explicit resource or even context[:work_package] and context[:project] would be good. I'll look into it.

Reply:
Update: Unfortunately I did not find any usage with a `WorkPackage` in `context[:object]`. Work package list and work package fullscreen views all call the API, which is using the Wrapper. 
I therefor added tests to make sure the formatter is working as expected either way. Adding a more explicit `context[:resource]` or `context[:work_package]` would require a lot of code changes, which would foil the purpose of avoiding the checks in the formatter.